images/ubuntu/templates/locals.ubuntu.pkr.hcl

```diff
@@ -0,0 +1,21 @@
+locals {
```

Comment:
Will it be better if we pass `image_os` to packer instead of `image_sku`? And we will choose `image_sku` here based on `image_os`. With this approach, we will be able to move hardcoded skus from `build-image.ps1` to `locals.ubuntu.pkr.hcl` and `locals.windows.pkr.hcl`